### PR TITLE
hw2 rebase cdw2146

### PR DESCRIPTION
**Pull request by chriswebb09**
added in readme link, uni and project contributions

added in descriptions in project .md files

added in details to main readme, update the descriptions for flask and swift docs under projects

### CODE REVIEW

File: cdw2146.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Name: Chris Webb
+
+UNI: cdw2146
+
+GitHub: [link](https://github.com/chriswebb09)
+
+GitHub README: [link](https://github.com/chriswebb09/chriswebb09/blob/main/README.md)
+
+# Project 
+[DirectReport](https://github.com/chriswebb09/DirectReport) - Keep track of your accomplishments each day of the workweek, create a report of things you accomplished at the end of the week that you can email to manager. Review progress each quarterly for more effective performance review.
```
Comment (timkpaine) [inline cdw2146.md:12]: Missing markdown file for project

File: cdw2146.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+GitHub README: [link](https://github.com/chriswebb09/chriswebb09/blob/main/README.md)
+
+# Project 
+[DirectReport](https://github.com/chriswebb09/DirectReport) - Keep track of your accomplishments each day of the workweek, create a report of things you accomplished at the end of the week that you can email to manager. Review progress each quarterly for more effective performance review. 
+
+Python / Javascript / Commandline / Web Interface 
+
+# Project Contributions
+
+* [Swift](https://github.com/apple/swift) C++ and Swift
```
Comment (timkpaine) [inline cdw2146.md:18]: Should link to markdown files

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: cdw2146.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+# [Project](https://github.com/chriswebb09/DirectReport)
+[DirectReport](https://github.com/chriswebb09/DirectReport) - Keep track of your accomplishments each day of the workweek, create a report of things you accomplished at the end of the week that you can email to manager. Review progress each quarterly for more effective performance review. 
+
+[README](https://github.com/chriswebb09/DirectReport/blob/main/README.md)
+
+
+
+# [Project Contributions](./projects)
+
+* [Swift](https://github.com/apple/swift) C++ and Swift - [README](./projects/swift/swift.md)
```
Comment (timkpaine) [inline cdw2146.md:20]: these paths are all off

File: cdw2146.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+GitHub README: [link](https://github.com/chriswebb09/chriswebb09/blob/main/README.md)
+
+# [Project](https://github.com/chriswebb09/DirectReport)
+[DirectReport](https://github.com/chriswebb09/DirectReport) - Keep track of your accomplishments each day of the workweek, create a report of things you accomplished at the end of the week that you can email to manager. Review progress each quarterly for more effective performance review. 
+
+[README](https://github.com/chriswebb09/DirectReport/blob/main/README.md)
+
+
+
+# [Project Contributions](./projects)
```
Comment (timkpaine) [inline cdw2146.md:18]: unnecessary links

File: cdw2146.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+* [Swift](https://github.com/apple/swift) C++ and Swift - [README](./projects/swift/swift.md)
+* [Typescript Website](https://github.com/microsoft/TypeScript-Website) Typescript - [README](./projects/javascript/typescript.md)
+* [Sourcery](https://github.com/krzysztofzablocki/Sourcery) Swift - [README](./projects/swift/sourcery.md)
+* [Flask](https://github.com/pallets/flask) Python - [README](./projects/python/flask.md)
+* 
+[JavaScript](./projects/javascript) / [Python](./projects/python) / [Swift](./projects/swift)
```
Comment (timkpaine) [inline cdw2146.md:25]: unnecessary links

File: cdw2146.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+[README](https://github.com/chriswebb09/DirectReport/blob/main/README.md)
+
+
+
+# [Project Contributions](./projects)
+
+* [Swift](https://github.com/apple/swift) C++ and Swift - [README](./projects/swift/swift.md)
+* [Typescript Website](https://github.com/microsoft/TypeScript-Website) Typescript - [README](./projects/javascript/typescript.md)
+* [Sourcery](https://github.com/krzysztofzablocki/Sourcery) Swift - [README](./projects/swift/sourcery.md)
```
Comment (timkpaine) [inline cdw2146.md:22]: no `sourcery.md` included

**Review by timkpaine — CHANGES_REQUESTED** [review]
squash all commits into 1

File: cdw2146.md (hunk 5 above)
Comment (chriswebb09) [reply cdw2146.md]: Removed

File: cdw2146.md (hunk 4 above)
Comment (chriswebb09) [reply cdw2146.md]: Removed

File: cdw2146.md (hunk 6 above)
Comment (chriswebb09) [reply cdw2146.md:22]: Added

File: cdw2146.md (hunk 3 above)
Comment (chriswebb09) [reply cdw2146.md]: I tested them it seems to work. I'm only tracking Swift as a project now, not by language.

**Comment by chriswebb09** [comment]
@timkpaine closing this out, it went awry with the amount of stuff needed to merge will reopen  =/

**Pull request closed without merging**